src/queries/types/GetAdminNotesAndActions.ts

```diff
@@ -27,12 +27,12 @@ export interface GetAdminNotesAndActions_systemIntake_actions_lcidExpirationChan
   __typename: "SystemIntakeLCIDExpirationChange";
   previousDate: Time;
   newDate: Time;
-  previousScope: string;
-  newScope: string;
-  previousNextSteps: string;
-  newNextSteps: string;
-  previousCostBaseline: string;
-  newCostBaseline: string;
+  previousScope: string | null;
```

Comment:
With the updates to these types, `src/views/GovernanceReviewTeam/Notes/index.tsx` (which uses this query) needs to be updated to account for possibly `null` text.

Reply:
Hmm what is the best way of dealing with this? Right now I just have a check for the fields like this

```
{lcidExpirationChange.previousNextSteps ? (
    display next steps
) : (
    display "No Next Steps Specified"
)
```

Any thoughts on that? Feels slightly bad because there should theoretically always be a scope and next steps unlike cost baseline, maybe display a slightly different message making it more clear the data was bad?

Reply:
Could probably be made a bit more succinct like

```jsx
text={lcidExperiationChange.previousNextSteps || t('notes.noneSpecified')}
```

Reply:
Looks good with https://github.com/CMSgov/easi-app/pull/1440/commits/945d7327e212c37bb65ea65284e5a8649ff13323.